File: src/main/java/me/wuzzyxy/dynamicmarket/commands/args/ForcePushCommand.java
package me.wuzzyxy.dynamicmarket.commands.args;

import me.wuzzyxy.dynamicmarket.market.MarketDatabaseHandler;
import org.bukkit.command.CommandSender;

import java.util.Optional;

public class ForcePushCommand implements ArgsCommand{
    MarketDatabaseHandler handler;
    public ForcePushCommand(MarketDatabaseHandler handler) {
        this.handler = handler;
    }
    @Override
    public Optional<String[]> execute(String[] args, CommandSender sender) {
        handler.pushItems();
        return Optional.of(new String[]{"Pushed items to database"});
    }
}

File: src/main/kotlin/me/wuzzyxy/dynamicmarket/commands/args/ForcePushCommand.kt
package me.wuzzyxy.dynamicmarket.commands.args

import me.wuzzyxy.dynamicmarket.market.MarketDatabaseHandler
import org.bukkit.command.CommandSender

class ForcePushCommand(private val handler: MarketDatabaseHandler) : ArgsCommand {

    override fun execute(args: Array<out String>, sender: CommandSender): List<String> {
        handler.pushItems()
        return listOf("Pushed items to database")
    }
}
